src/components/AddressInputPanel/index.tsx

```diff
@@ -99,7 +105,7 @@ export default function AddressInputPanel({
           <AutoColumn gap="md">
             <RowBetween>
               <TYPE.black color={theme.text2} fontWeight={500} fontSize={14}>
-                Recipient
+                {label ?? 'Recipient'}
```

Comment:
might be simpler to just default the value to recipient in the params

`AddressInputPanel({id, className= '...', label = 'Recipient', ...})`